pkg/volume/util/util.go

```diff
@@ -545,6 +545,6 @@ func GetPluginMountDir(host volume.VolumeHost, name string) string {
 // volume vs. some other type
 func IsLocalEphemeralVolume(volume v1.Volume) bool {
 	return volume.GitRepo != nil ||
-		(volume.EmptyDir != nil && volume.EmptyDir.Medium != v1.StorageMediumMemory) ||
+		(volume.EmptyDir != nil && volume.EmptyDir.Medium == v1.StorageMediumDefault) ||
 		volume.ConfigMap != nil || volume.DownwardAPI != nil
```

Comment:
It seems like now some checks that didn't have `volume.DownwardAPI != nil` will now have that check. Could that have adverse impacts? What exactly is `volume.DownwardAPI` and when could it be nil?

Reply:
I found that `volume.DownwardAPI` is mounted as tmpfs. It is not right treating volume.Downward API as local ephemeral volume, so I deleted this condition.
It is mentioned [here](https://github.com/kubernetes/kubernetes/pull/55447#discussion_r152144318).